frontend/src/pages/questionList/subComponents/QuestionTableRow.jsx

```diff
@@ -32,19 +62,59 @@ export default function QuestionTableRow({ question }) {
         displayAlert('error', `Failed to get user questions. Error: ${err}`);
       }
     }
-    // Get all answers for selected question
-    const response = await API.cache.getAnswersByQuestion(question.id, accessToken);
+    if (question.metadata.answerOnly) {
+      // This is an answer with no parent, send to
+      // answer page
+      history.push(`/answer/${question.id}`);
+    } else {
+      // Get all answers for selected parent question
+      const response = await API.cache.getAnswersByQuestion(question.id, accessToken);
+      if (response.status === 'error') {
+        displayAlert('error', 'Failed to load answers.');
+        return;
+      }
+
+      if (response.length) {
+        // redirect to first answer
+        const answerId = response[0].id;
+        history.push(`/answer/${answerId}`);
+      } else {
+        displayAlert('warning', 'There are no answers for this question.');
+      }
+    }
+  }
+
+  async function deleteQuestion() {
+    let accessToken;
+    if (isAuthenticated) {
+      try {
+        accessToken = await getAccessTokenSilently();
+      } catch (err) {
+        displayAlert('error', `Failed to get user questions. Error: ${err}`);
```

Comment:
```suggestion
        displayAlert('error', `Failed to get authenticate user. Error: ${err}`);
```

Reply:
Updated in 06dfb063189bcbd4f9ce23d3ace08fba1c9301f6